COLLABORATOR_GUIDE.md

```diff
@@ -111,3 +118,112 @@ layout: contribute.hbs
 ### Translating pages
 
 See [TRANSLATION.md](./TRANSLATION.md) for the website translation policy.
+
+## Creating Components
+
+The Node.js Website uses **React.js** as a Frontend Library for the development of the Website. React allows us to create user interfaces with a modern take on Web Development.
+
+If you're unfamiliar with React or Web Development in general, we encourage a read before taking on complex issues and tasks as this repository is **not for educational purposes** and we expect you to have a basic understanding of the technologies used.
+
+We also recommend getting familiar with technologies such as [Next.js][], [MDX][], [SCSS][] and "concepts" such as "CSS Modules" and "CSS-in-JS".
```

Comment:
Could you please fix the reference style links here?

They can likely be removed from https://github.com/nodejs/nodejs.org/pull/5416/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R206-R208 and added to the bottom of this file instead

Reply:
Thank you @mikeesto - closed this one in favor of https://github.com/nodejs/nodejs.org/pull/5420 changes should be reflected there!